app/src/main/java/org/fossasia/pslab/fragment/LALogicLinesFragment.java

```diff
@@ -390,7 +397,9 @@ public void onNothingSelected() {
 
     }
 
-    private void updateLogicLines(double[] xData, double[] yData) {
+    // Single Channel Mode
```

Comment:
Can we put this comment closer to the method you are referring to?

Reply:
This comment is common for all the three methods. There will be more such comments for every channel mode methods

Reply:
Yes, my point is not comments. there is an empty line. Please remove it.